who_l3_smart_tools/core/terminology/who/schema.py

```diff
@@ -1,11 +1,11 @@
 """
-This module defines HIVschema classes for an OCL Concepts, 
+This module defines HIVschema classes for an OCL Concepts,
     Repository and Organizations.
 """
 
 from typing import Callable
-from who_l3_smart_tools.core.terminology.schema import ConceptSchema
-from who_l3_smart_tools.utils.terminology import dash_preserving_slugify
+from ..schema import ConceptSchema
+from ....utils import dash_preserving_slugify
```

Comment:
This is a good example of how these can quickly become hard to read. It is not as clear to know where this is from than if it was imported with the absolute path

Reply:
To what extent is that an important consideration?